Samples/Client/Client_Subscribe_Samples.cs

```diff
@@ -177,4 +177,60 @@ public static async Task Subscribe_Topic()
             response.DumpToConsole();
         }
     }
-}
+
+    public static void ConcurrentProcessingWithLimit(CancellationToken shutdownToken, IMqttClient mqttClient)
```

Comment:
Please apply snake case here for the method name because it gets printed to the console. All other methods are using snake case in order to improve reading in the console.